x/accounts/cli/cli_test.go

```diff
@@ -0,0 +1,150 @@
+package cli_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"testing"
+
+	abci "github.com/cometbft/cometbft/api/cometbft/abci/v1"
+	rpcclientmock "github.com/cometbft/cometbft/rpc/client/mock"
```

Comment:
are we able to not add a direct cometbft/cometbft dependency in x/accounts just for tests?

Reply:
good catch, missed this

Reply:
hmmm let me see if i can create mock for rpc client

Reply:
add a small hack for it